neural_speed/core/ne_layers.c

```diff
@@ -9040,6 +9047,38 @@ static void ne_compute_forward_rope_f32(const struct ne_compute_params* params,
             dst_data[n_dims] = x2 * cos_block_theta - x3 * sin_block_theta;
             dst_data[n_dims / 2 * 3] = x2 * sin_block_theta + x3 * cos_block_theta;
           }
+        } else if (is_longrope) {
+          // TODO: this is probably wrong, but I can't figure it out ..
```

Comment:
this is probably wrong, but I can't figure it out
what is the mean of this ？

Reply:
removed